docs_src/microservices/application/AdvancedTopics.md

```diff
@@ -424,3 +424,57 @@ If in secure mode, the secrets are retrieved from the secret store based on the
 
 If running in insecure mode, the secrets are retrieved from the *Writable.InsecureSecrets* configuration.
 
+### Background Publishing
+
+Application Services using the MessageBus trigger can request a background publisher using the AddBackgroundPublisher API in the SDK.  This method takes an int representing the background channel's capacity as the only parameter and returns a reference to a BackgroundPublisher.  This reference can then be used by background processes to publish to the configured MessageBus output.  Example usage is as follows:
+
+```go
+
+func runJob (pub appsdk.BackgroundPublisher, done chan struct{}){
+	ticker := time.NewTicker(1 * time.Minute)
+	go func() {
+		for {
+			select {
+			case <-ticker.C:
+				msg := myDataService.GetMessage()
+				payload, err := json.Marshal(message)
+				
+				if err != nil {
+					continue
+				}
+				pub.Publish(, fmt.Sprintf("background=%d", time.Now().UnixNano()), clients.ContentTypeJSON)
```

Comment:
Missing `payload` as first parameter.
Also recommend set `correlationId = uuid.New().String() ` and passing it to `Publish` so it is more explicit and uses standard UUID used else where for correlation ID.

Reply:
Doh should be good now